pss/pss.go

```diff
@@ -27,12 +27,21 @@ import (
 // Pss is the top-level struct, which takes care of message sending
 type Pss struct {
 	localStore chunk.Store
+	tag        *chunk.Tag
+}
+
+// State exposes underling chunk states
+type State struct {
```

Comment:
i don't understand what the benefit of defining this type is when we are just importing the `chunk.State` values anyway. plus `tag.go` already exposes these.

clarification would be appreciated

Reply:
this was used when the tag was in the struct, This was an iteration.
removed